repo/packages/B/beta-tensorflow/2/config.json

```diff
@@ -0,0 +1,410 @@
+          "type": "string",
+          "default": ""
+        },
+        "job_name": {
+          "title": "Job Name",
+          "description": "Name of the module (without extension) whose main function should be executed.",
+          "type": "string",
+          "default": ""
+        },
+        "job_context": {
+          "title": "Job Context (Optional)",
+          "description": "A dictionary to pass to your job's main function (intended for run-time configuration options like batch_size, learning_rate, etc.).",
+          "type": "string",
+          "default": "{}"
+        },
+        "shared_filesystem": {
+          "title": "Shared Filesystem",
+          "description": "Distributed TensorFlow relies on a shared filesystem (GCS, HDFS, etc.) for summaries/checkpoints. Please set up one of these filesystems before using this package, and specify the location here (eg. 'gs://your/directory' or 'hdfs://default/your/directory').",
+          "type": "string",
+          "default": ""
+        },
+        "hdfs": {
+          "type": "object",
+          "description": "HDFS properties",
+          "properties": {
+            "config_uri": {
+              "title": "HDFS Configuration Directory URI",
+              "description": "URI to parent directory for HDFS configuration files 'hdfs-site.xml' and 'core-site.xml'",
+              "type": "string",
+              "default": ""
+            }
+          }
+        },
+        "gcs_key": {
+          "title": "GCS Key",
+          "description": "Stringified contents of JSON key for Google Cloud Storage service account. Only use this field if use_gcs_key_secret is set to False.",
+          "type": "string",
+          "default": ""
+        },
+        "use_gcs_key_secret": {
+          "description": "A DC/OS secret called gcs_key exists and should be consumed by this framework.",
+          "type": "boolean",
+          "default": false
+        },
+        "use_tensorboard": {
+          "description": "Should the framework also launch an instance of Tensorboard? Make sure you have a shared filesystem set up first! You can still run Tensorboard locally if this is set to false",
+          "type": "boolean",
+          "default": false
+        },
+        "tf_image": {
+          "title": "TensorFlow Docker Image",
+          "description": "Name of the TensorFlow Docker Image in which Worker/PS tasks will be run (usr/img:version).",
+          "type": "string",
+          "default": "mesosphere/dcos-tensorflow:0.0.3-1.5.0"
+        },
+        "gpu_tf_image": {
+          "title": "TensorFlow GPU Docker Image",
+          "description": "Name of the TensorFlow Docker Image in which GPU Worker tasks will be run (usr/img:version).",
+          "type": "string",
+          "default": "mesosphere/dcos-tensorflow:0.0.3-1.5.0-gpu"
```

Comment:
@elezar Is there a reason we moved from default value of mesosphere/dcos-tensorflow:2.7.4-1.3.0 to mesosphere/dcos-tensorflow:0.0.3-1.5.0-gpu?

Reply:
We don't bundle hdfs anymore, and as such the 0.0.3 is just a sequence number. The Tensorflow version was changed to 1.5.0.